classes/MiniSurvey.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace MySociety\TheyWorkForYou;
+
+require_once INCLUDESPATH . "../../commonlib/phplib/random.php";
+require_once INCLUDESPATH . "../../commonlib/phplib/auth.php";
+
+/**
+ * Hansard
+ */
+
+class MiniSurvey {
+
```

Comment:
Could the old style code get data and output using this class too, to save on duplication?